roles/jenkins_slave/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+- name: install rpm packages
+  yum:
+    name: "{{ jslave_rpm_deps | join(',') }}"
+    state: present
+
+- name: install pip packages
```

Comment:
Definitely separate issue type of work. It would be best, I would think, to put this into a virtualenv instead of into the system root, along with the two packages below this one.